pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go

```diff
@@ -1,31 +1,31 @@
 package migrations
 
-import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+// This migration was run behind the `dashboardPreviews` flag. The feature flag and the whole dashboard previews feature was removed in https://github.com/grafana/grafana/pull/66176
```

Comment:
I would remove these migrations completely since they are independent anyway.

Reply:
I was concerned that having records in the migrations table that refer to migrations that don't exist might be problematic but it seems that the migrator doesn't actually care about that.  If anything we might want to add a migration to drop the table if it does exist.

Reply:
thanks both - in the next PR ill either remove this file, or add a migration to remove the table if thats easy enough